src/test/kotlin/step1/PersonTest.kt

```diff
@@ -0,0 +1,55 @@
+package step1
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+
+class PersonTest {
+    @Test
+    fun `named arguments`() {
+        val people =
+            listOf(
+                Person(name = "Karyna", age = 20, nickname = "sh1erz"),
+                Person(age = 20, nickname = "sh1erz", name = "Karyna"),
+                Person(nickname = "sh1erz", name = "Karyna", age = 20),
+            )
+        assertThat(people).allSatisfy {
```

Comment:
What assertion libraries do you often use for verification? I personally use AssertJ and Kotest quite a bit, and getting familiar with their APIs helps me write better test code.